ci_framework/roles/edpm_deploy/tasks/main.yml

```diff
@@ -19,11 +19,26 @@
     - always
   ansible.builtin.set_fact:
     cifmw_edpm_deploy_env: "{{ cifmw_install_yamls_environment | combine({'PATH': cifmw_path }) }}"
+    cifmw_edpm_deploy_provisioned_compute_env: |
+      {% if cifmw_external_compute_info is defined %}
```

Comment:
ok, so you're expecting the "parameters" directory to be kind of reloaded - I'm not sure this is actually the case.

In case we have to rely on the hook, it would be safer to rely on its embedded mechanism to load generated parameter file.